bears/natural_language/LanguageToolBear.py

```diff
@@ -53,8 +55,12 @@ def run(self,
         # language_check being there.
         from language_check import LanguageTool, correct
 
+        for requirement in list(self.__class__.REQUIREMENTS):
```

Comment:
https://gitlab.com/coala/package_manager/merge_requests/91 has landed.
I'll get it released and available for you today.
But you can already experiment with it (e.g. by changing the versions manually).
I'd love to see your patch merged this weekend if possible, so I can finish #1000.